lib/Controller/OutboxController.php

```diff
@@ -209,6 +209,9 @@ public function update(
 		?int $sendAt = null
 	): JsonResponse {
 		$message = $this->service->getMessage($id, $this->userId);
+		if ($message->getStatus() === LocalMessage::STATUS_IMAP_SENT_MAILBOX_FAIL || $message->getStatus() === LocalMessage::STATUS_PROCESSED) {
+			return JsonResponse::error('Cannot modify already sent message', Http::STATUS_FORBIDDEN, [$message]);
```

Comment:
```suggestion
			return JsonResponse::error('Cannot modify already sent message', Http::STATUS_FORBIDDEN, $message);
```

Reply:
I don't think that's right?

https://github.com/nextcloud/mail/blob/3285422eef37b9716cb31aeb66b76dcacd5ef3b2/lib/Controller/OutboxController.php#L150

Reply:
`error::$data` ist typed to `[]` unlike `success` which has no enforced type